src/components/CompareResults/SubtestsResults/SubtestsResultsTable.tsx

```diff
@@ -0,0 +1,259 @@
+};
+
+function processResults(results: CompareResultsItem[]) {
+  const processedResults: Map<string, CompareResultsItem[]> = new Map<
+    string,
+    CompareResultsItem[]
+  >();
+  results.forEach((result) => {
+    const { new_rev: newRevision, header_name: header } = result;
+    const rowIdentifier = header.concat(' ', newRevision);
+    if (processedResults.has(rowIdentifier)) {
+      (processedResults.get(rowIdentifier) as CompareResultsItem[]).push(
+        result,
+      );
+    } else {
+      processedResults.set(rowIdentifier, [result]);
+    }
+  });
+  const restructuredResults: SubtestsResults[] = Array.from(
+    processedResults,
+    function ([rowIdentifier, result]) {
+      return {
+        key: rowIdentifier,
+        value: result,
+        // revisionHeader: {
+        //   suite: result[0].suite,
+        //   framework_id: result[0].framework_id,
+        //   test: result[0].test,
+        //   option_name: result[0].option_name,
+        //   extra_options: result[0].extra_options,
+        //   new_rev: result[0].new_rev,
+        //   new_repo: result[0].new_repository_name,
+        // },
+      };
+    },
+  );
+
+  return restructuredResults;
+}
+
+const cellsConfiguration: CompareResultsTableConfig[] = [
+  {
+    name: 'Subtests',
+    // disable: true,
+    // filter: true,
+    key: 'subtests',
+    // possibleValues: ['Windows', 'OSX', 'Linux', 'Android'],
+    // matchesFunction: (result: CompareResultsItem, value: string) => {
+    //   const platformName = getPlatformShortName(result.platform);
+    //   return platformName === value;
+    // },
+  },
+  {
+    name: 'Base',
+    key: 'base',
+  },
+  { key: 'comparisonSign' },
+  { name: 'New', key: 'new' },
+  {
+    name: 'Status',
+    disable: true,
+    filter: true,
+    key: 'status',
+    possibleValues: ['No changes', 'Improvement', 'Regression'],
+    matchesFunction: (result: CompareResultsItem, value: string) => {
+      switch (value) {
+        case 'Improvement':
+          return result.is_improvement;
+        case 'Regression':
+          return result.is_regression;
+        default:
+          return !result.is_improvement && !result.is_regression;
+      }
+    },
+  },
+  {
+    name: 'Delta(%)',
+    key: 'delta',
+  },
+  {
+    name: 'Confidence',
+    disable: true,
+    filter: true,
+    key: 'confidence',
+    possibleValues: ['Low', 'Medium', 'High'],
+    matchesFunction: (result: CompareResultsItem, value: string) =>
+      result.confidence_text === value,
+  },
+  { name: 'Total Runs', key: 'runs' },
+  { key: 'buttons' },
+  { key: 'expand' },
+];
+
+function resultMatchesSearchTerm(
+  result: CompareResultsItem,
+  searchTerm: string,
+) {
+  return (
+    result.suite.includes(searchTerm) ||
+    result.extra_options.includes(searchTerm) ||
+    result.option_name.includes(searchTerm) ||
+    result.test.includes(searchTerm) ||
+    result.new_rev.includes(searchTerm) ||
+    result.platform.includes(searchTerm)
```

Comment:
```suggestion
    result.test.includes(searchTerm)
```

Reply:
Since all the other fields are the same for each result except the test name then I suggest excluding them from the search functionality.